wagtailio/static/sass/abstracts/_mixins.scss

```diff
@@ -28,3 +28,12 @@
     white-space: nowrap;
     border: 0;
 }
+
+// Streamfield spacing
+@mixin sf-spacing() {
```

Comment:
**Praise:** Noice! Definitely needed this

Reply:
We could use this as a block spacer throughout other parts of the site as well (not just for streamfields). Perhaps in the future we can add params for specifying top or bottom spacing or both for different scenarios. Not required as part of this work though 😅

Reply:
That sounds useful. The streamfields had pretty consistent spacing in the design but no reason why we couldn't rename and adopt this for other components - would be nice to have it all controlled from one source : )